src/Magento/FunctionalTestingFramework/Module/MagentoPwaWebDriver.php

```diff
@@ -17,16 +17,15 @@ class MagentoPwaWebDriver extends MagentoWebDriver
 {
     /**
      * Wait for a PWA Element to NOT be visible using JavaScript.
+     * Add the WAIT_TIMEOUT variable to your .env file for this action.
      *
      * @param null $selector
      * @param null $timeout
      * @throws \Exception
      * @return void
      */
-    public function waitForPwaElementNotVisible($selector, $timeout = null)
+    public function waitForPwaElementNotVisible($selector, $timeout)
```

Comment:
This is not backward compatible change.  Has the previous code been released? I think it's ok to leave it unchanged. Could you please confirm?

Reply:
Reverted this change.